share/hooks/raft/vip.sh

```diff
@@ -17,7 +17,7 @@ fi
 case $ACTION in
 leader)
     sudo ip address add $IP dev $INTERFACE
-    arping -c 5 -U -I $INTERFACE ${IP%%/*}
+    sudo arping -c 5 -U -I $INTERFACE ${IP%%/*}
```

Comment:
I'm running on arch and there is no setuid flag on arping so i added the sudo as it's already in for the ip command